js/block.js

```diff
@@ -4377,6 +4377,10 @@ function Block(protoblock, blocks, overrideName) {
         };
 
         this._exitWheel.navItems[1].navigateFunction = function () {
+            if(that.value == 1) {
```

Comment:
why == 1??

Reply:
Also, this pie menu is used for many other situations where negative numbers are OK.

Reply:
I have changed this in the latest commit to prevent the value from going below zero if number block is connected to a note block.